persistence/src/main/scala/hmda/persistence/demo/DemoData.scala

```diff
@@ -85,6 +85,7 @@ object DemoData {
       case (instId: String, filings: Seq[Filing]) =>
         val filingActor = system.actorOf(FilingPersistence.props(instId))
         filings.foreach { filing => filingActor ? CreateFiling(filing) }
+        Thread.sleep(100)
```

Comment:
Actually, I tested again locally, and I'm no longer seeing the errors. I'll update to remove that `Thread.sleep`.